ocaml/tests/test_vm_helpers.ml

```diff
@@ -569,3 +569,10 @@ let test =
     , `Quick
     , test_get_group_key_vgpu_and_netsriov )
   ]
+
+let () =
+  Suite_init.harness_init () ;
+  (* Alcotest hides the standard output of successful tests,
+     so we will probably not exceed the 4MB limit in Travis *)
+  Debug.log_to_stdout () ;
+  Alcotest.run "Test VM Helpers suite" ["test", test]
```

Comment:
You could add a helper to avoid repeating this pattern, something like:

```
let run_with_init name test =
   Suite_init.harness_init () ;
  (* Alcotest hides the standard output of successful tests,
     so we will probably not exceed the 4MB limit in Travis *)
  Debug.log_to_stdout () ;
  Alcotest.run name ["test", test]
```